tests/modules/kat/hist/test.yml

```diff
@@ -0,0 +1,30 @@
+- name: kat hist test_kat_hist_single_end
+  command: nextflow run tests/modules/kat/hist -entry test_kat_hist_single_end -c tests/config/nextflow.config
+  tags:
+    - kat/hist
+    - kat
+  files:
+    - path: output/kat/test.hist
+      md5sum: 1aceb823b6774f14f5cca15954138efd
+    - path: output/kat/test.hist-hash.jf27
+    - path: output/kat/test.hist.dist_analysis.json
+      md5sum: ec4317d510f752855411d13c0f24dea9
+    - path: output/kat/test.hist.png
+      md5sum: 30daaf4d2dd4b1574e387df973c39d03
+    - path: output/kat/versions.yml
+      md5sum: 296f28c007bc55f8f5490702cf3f81d4
```

Comment:
I think we have agreed to avoid md5sums for versions. Julia already added this to nf-core/tools dev.

Reply:
👍🏽 I'll run the dev version then.

Reply:
could also just delete that line

Reply:
true
```suggestion
```